File: DAYONE/Python/NumberGuessingApp.py
import random

count = 0; 
user_input = 0;
random_guess = random.randrange(101)

while(True):
	user_input = int(input("Enter a number: "))

	if (user_input < 1 or user_input > 100):
		print("Please enter a number between 1 and 100");
		continue;
	else:
		count += 1;
		if (random_guess < user_input):
			print("lower - Try again")

		if (random_guess > user_input) :
			print("Higher - Try again")

		if (random_guess == user_input or count == 5):
			break;



print("============FINAL SUMMARY============");
print("The correct number is: " , random_guess);
print("The number of attempts is: " , count);

if (random_guess == user_input and count == 1):
	print("your rating is: Legendary");
elif (random_guess == user_input and count == 2):
	print("your rating is: Excellent");
elif (random_guess == user_input and count == 3 or count == 4):
	print("your rating is: Good");
elif (random_guess == user_input and count == 5):
	print("your rating is: Close!");
else :
	print("No win = Better luck");


print("=====================================");

File: DAYONE/Java/NumberGuessingApp.java
import java.util.*;

public class NumberGuessingApp {
public static void main(String... azeez) {
Scanner input = new Scanner(System.in);
Random rand = new Random();

int randomGuess = rand.nextInt(100);
int count = 0; 
int userInput = 0;

while(true){
System.out.println("Enter a number");
userInput = input.nextInt();

if (userInput < 1 || userInput > 100){
System.out.println("Please enter a number between 1 and 100");
continue;
}
else{
count += 1;
if (randomGuess < userInput) {
System.out.println("lower - Try again");
}
if (randomGuess > userInput) {
System.out.println("Higher - Try again");
}
if (randomGuess == userInput || count == 5){
break;
}
}
}
System.out.println("============FINAL SUMMARY============");
System.out.println("The correct number is: " + randomGuess);
System.out.println("The number of attempts is: " + count);

if (randomGuess == userInput && count == 1) System.out.println("your rating is: Legendary");
else if (randomGuess == userInput && count == 2) System.out.println("your rating is: Excellent");
else if (randomGuess == userInput && count == 3 || count == 4) System.out.println("your rating is: Good");
else if (randomGuess == userInput && count == 5) System.out.println("your rating is: Close!");
else {
System.out.println("No win = Better luck");
}
System.out.println("=====================================");
}
}

